src/D2L.CodeStyle.Analyzers/Extensions/Microsoft.CodeAnalysis.CSharp.cs

```diff
@@ -17,7 +17,8 @@ internal static partial class RoslynExtensions {
 		public static IParameterSymbol DetermineParameter(
 			this ArgumentSyntax argument,
 			SemanticModel semanticModel,
-			bool allowParams = false

```

Comment:
Only one caller was using the default value of `false`, so pushed it to the caller